iDiff/tests/multi_hist_diff_expected.json

```diff
@@ -5,7 +5,7 @@
       "Image1": "gcr.io/gcp-runtimes/pip-base",
       "Image2": "gcr.io/gcp-runtimes/pip-modified",
       "Adds": [
-        "/bin/bash"
+        "/bin/bash\n"
```

Comment:
We should definitely handle this for the history differ ( @cftorres ) but for now I'm actually going to just remove this file (and the hist_diff_exepected.json) because we don't need them until we are running those tests again.